video_creation/final_video.py

```diff
@@ -7,80 +11,86 @@
     CompositeAudioClip,
     CompositeVideoClip,
 )
-import reddit.subreddit 
-import re
-from utils.console import print_step
-from dotenv import load_dotenv
-import os
 
-W, H = 1080, 1920
```

Comment:
Thought process behind removing these variables? Surely it's more useful to keep them in.